components/infobox/wikis/valorant/infobox_league_custom.lua

```diff
@@ -0,0 +1,210 @@
+local Variables = require('Module:Variables')
+local Tier = require('Module:Tier')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+local PageLink = require('Module:Page')
+local PrizePoolCurrency = require('Module:Prize pool currency')
+local RIOT_SPONSORED = '[[File:Riot Games Tier Icon.png|x18px|link=Riot Games|Tournament supported by Riot Games.]]'
+
+local _TODAY = os.date('%Y-%m-%d', os.time())
+
+local _args
+local _league
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_league = league
+	_args = _league.args
+
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.getWikiCategories = CustomLeague.getWikiCategories
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:addCustomCells(widgets)
+	local args = _args
+	table.insert(widgets, Cell{
+		name = 'Teams',
+		content = {(args.team_number or '') .. (args.team_slots and ('/' .. args.team_slots) or '')}
+	})
+	table.insert(widgets, Cell{
+		name = 'Patch',
+		content = {CustomLeague:_createGameCell(args)}
+	})
+
+	return widgets
+end
+
+function CustomInjector:parse(id, widgets)
+	local args = _args
+	if id == 'customcontent' then
+		if String.isNotEmpty(args.map1) then
+			local maps = {}
+
+			for _, map in ipairs(_league:getAllArgsForBase(args, 'map')) do
+				table.insert(maps, tostring(CustomLeague:_createNoWrappingSpan(
+					PageLink.makeInternalLink({}, map)
+				)))
+			end
+			table.insert(widgets, Title{name = 'Maps'})
+			table.insert(widgets, Center{content = {table.concat(maps, '&nbsp;• ')}})
+		end
+	elseif id == 'prizepool' then
+		return {
+			Cell{
+				name = 'Prize pool',
+				content = {CustomLeague:_createPrizepool()}
+			},
+		}
+	elseif id == 'liquipediatier' then
+		local tierDisplay = CustomLeague:_createLiquipediaTierDisplay()
+		local class
+		if args['riot-sponsored'] == 'true' then
+			tierDisplay = (tierDisplay or '').. '&nbsp;' .. RIOT_SPONSORED
+			class = {'valvepremier-highlighted'}
+		end
+		return {
+			Cell{
+				name = 'Liquipedia Tier',
+				content = {tierDisplay},
+				classes = class
+			}
+		}
+	end
+	return widgets
+end
+
+function CustomLeague:addToLpdb(lpdbData, args)
+	lpdbData.maps = table.concat(_league:getAllArgsForBase(args, 'map'), ';')
+	lpdbData.publishertier = args['riot-sponsored']
+	lpdbData.participantsnumber = args.team_number
+	lpdbData.liquipediatiertype = args.liquipediatiertype
+	return lpdbData
+end
+
+
+function CustomLeague:_createGameCell(args)
```

Comment:
Rename the function and where it's called (eg to `_createPatchCell`), since it doesn't create the game cell as there isn't one.